crud/select.lua

```diff
@@ -312,9 +335,17 @@ function select_module.call(space_name, user_conditions, opts)
         utils.reverse_inplace(tuples)
     end
 
+    local filtered_space_format, err = utils.get_fields_format(iter.space_format, opts.fields)
+
```

Comment:
Remove empty line.

Such `err ~= nil` checks relate to previous line, so it's convenient way to keep them together.